### DISCUSSION TITLE
Can't preview picture when click the thumbnails in chat list at WeChat for mac

### DISCUSSION THREAD

**Discussion by sugarZ** [rustdesk/rustdesk#9190] [General]
### Bug Description

Normally, when the mouse clicks on an image in the chat list, an image preview window will pop up
When using rustdesk remote control, No preview image window appears after clicked image.
The image preview window as shown in the screenshot cannot be displayed.

### How to Reproduce

remote control from windows to mac

### Expected Behavior

click the thumbnail will popup a preview image window.

### Operating system(s) on local side and remote side

Windows 11->OSX

### RustDesk Version(s) on local side and remote side

1.30.0->1.30.0

### Screenshots

![xuanxuan_20240827_182843](https://github.com/user-attachments/assets/3895ef59-c814-4c72-b199-c4fb6116ec12)

**Comment by martint028** [comment]
Me too
